### ISSUE TITLE
Add support for the new Gatsby Image plugin

### ISSUE THREAD

**Issue by rshackleton** [cloudinary-devs/gatsby-transformer-cloudinary#90]
The plugin should be updated to support the new Gatsby Image plugin. The steps are outlined in the article below:

- https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/adding-gatsby-image-support/

**Comment by Chuloo** [comment]
Yes, this is being worked on 👍

**Comment by frodeste** [comment]
Will this allow us to use gatsby-plugin-image with gatsby-transformer-cloudinary? So that I can have all images in cloudinary only, and use gatsby to serve correct image to the browser?

**Comment by louishugens** [comment]
@Chuloo Hi. where are we on this?

**Comment by frodeste** [comment]
?

**Comment by raae** [comment]
@olavea and I are tasked with getting this plugin up-to-date 🥳
We'll be adding support for the new Gatsby Image plugin.

**Comment by raae** [comment]
We decided to got with the same options as the current version, and also keep support for gatsby-image for now to make the upgrade path easier. 

```
query {
      image: file(name: { eq: "jason" }) {
        cloudinary: childCloudinaryAsset {
          gatsbyImageData(
            width: 300
            layout: FIXED
            placeholder: BLURRED
            transformations: ["e_grayscale"]
            chained: ["t_lwj"]
          )
          fixed(
            width: 300
            transformations: ["e_grayscale"]
            chained: ["t_lwj"]
          ) {
            ...CloudinaryAssetFixed
          }
        }
      }
    }
```

**Comment by raae** [comment]
Who would be willing to help test this before merging to master?
@louishugens? @frodeste? @rshackleton?
Would be great if you had an existing site for a real test.

**Comment by frodeste** [comment]
I do not have any time now. Sorry

Med vennlig hilsen
Frode Stenstrøm | Prosjektdirektør | Prosjektkontoret
Mobil: +47 95704546<tel:+4795704546> |  E-post /Teams: ***@***.******@***.***>

**Comment by raae** [comment]
Check out the PR for testing procedures: #173

**Comment by cloudinary-raae** [comment]
Released: v2.3.0
